site/docs/platforms.md

```diff
@@ -178,7 +178,7 @@ FAILED: Build did NOT complete successfully
 
 For more flexibility in expressing constraints, use the
 `@platforms//:incompatible`
-[`constraint_value`](platform.html#constraint_value) that no platform
+[`constraint_value`](be/platforms.html#constraint_value) that no platform
```

Comment:
Isn't this the link we want? https://docs.bazel.build/versions/master/be/platform.html#constraint_value

(no "s" in `platform`)?